server/channels/app/opengraph.go

```diff
@@ -49,7 +52,12 @@ func (a *App) GetOpenGraphMetadata(requestURL string) ([]byte, error) {
 
 func (a *App) parseOpenGraphMetadata(requestURL string, body io.Reader, contentType string) *opengraph.OpenGraph {
 	og := opengraph.NewOpenGraph()
+
+	var buf bytes.Buffer
+	body2 := io.TeeReader(body, &buf)
```

Comment:
This is kinda confusing here. Maybe we should just read all here and create a `bytes.Reader` for `og.ProcessHTML`? We read all in `parseLinkFavicon` anyway.

Reply:
Done